.github/workflows/bc.yml

```diff
@@ -0,0 +1,10 @@
+on: [push, pull_request]
+name: Roave
+jobs:
+  roave_bc_check:
+    name: BC Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Roave BC Check
+        uses: docker://nyholm/roave-bc-check-ga
```

Comment:
Can this be versioned?

Reply:
It can, but why? I would like to have all the warnings possible. Even if roave BC check updates their major version.

Reply:
While indeed nice to have latest and greatest, tools like CS and SA would then break for random contributors jumping in

Reply:
I agree, pipelines should be as static as possible. What's working today, should work tomorrow as well. We need to update these versions regularly. Same for actions, like `actions/checkout@master`. It should be `@v1`.